api/src/main/java/com/github/retrooper/packetevents/protocol/player/User.java

```diff
@@ -84,6 +104,151 @@ public void setConnectionState(ConnectionState connectionState) {
         this.setEncoderState(connectionState);
     }
 
+    @Override
+    public Identity identity() {
+        return identity;
+    }
+
+    @Override
+    public <T> void sendTitlePart(@NotNull TitlePart<T> part, @NotNull T value) {
+        Component title = null, subtitle = null;
+        Title.Times times = null;
+        if (part == TitlePart.TITLE) {
+            title = (Component) value;
+        }
+        else if (part == TitlePart.SUBTITLE) {
+            subtitle = (Component) value;
+        }
+        else if (part == TitlePart.TIMES) {
+            times = (Title.Times) value;
+        }
+        int stay, out, in;
+        if (times != null) {
+            stay = (int) (times.stay().toMillis() / 20);
+            out = (int) (times.fadeOut().toMillis() / 20);
+            in = (int) (times.fadeIn().toMillis() / 20);
+        }
+        else {
+            stay = 20;
+            out = 20;
+            in = 20;
+        }
+        sendTitle(title, subtitle, in, stay, out);
```

Comment:
are you sure this is what sendTitlePart is supposed to do? wont this override the title, if you send a title-part and then a subtitle-part?

Reply:
> are you sure this is what sendTitlePart is supposed to do? wont this override the title, if you send a title-part and then a subtitle-part?

It would override yeah